### PR TITLE
Added Searching Functionality

### PR DESCRIPTION
**Pull request by jainaman224**
Added Searching Functionality to `speakers` and `rooms` page. @mariobehling @SaptakS please review.

### CODE REVIEW

**Comment by mariobehling** [comment]
@jainaman224 Thank you. Would be great, if you could solve the hound-ci issues in this PR. Cheers.

**Comment by mariobehling** [comment]
Ok, thank you. Merging it. Let's improve remaining issues step by step. Cheers.

File: src/backend/_scss/application.scss [hunk 1]
```diff
@@ -1302,3 +1302,14 @@ a {
...
+
+  #fossasia-filter {
+      font-family: verdana;
+      font-size: 10pt;
+      height: 28px;
+      width: 186px;
+      color: #000000;
+      padding-left: 2px;
+      border: 1px solid #696969;
+      background-color: #ffffff;
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: @jainaman224 Please refer config.scss for color coding.

File: src/backend/_scss/application.scss [hunk 2]
```diff
@@ -1302,3 +1302,14 @@ a {
...
   }
+
+  #fossasia-filter {
+      font-family: verdana;
+      font-size: 10pt;
+      height: 28px;
+      width: 186px;
+      color: #000000;
+      padding-left: 2px;
+      border: 1px solid #696969;
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: @jainaman224 Please refer config.scss for color coding.

File: src/backend/_scss/application.scss [hunk 3]
```diff
@@ -1302,3 +1302,14 @@ a {
   .graytext {
     color: $gray-dark;
   }
+
+  #fossasia-filter {
+      font-family: verdana;
+      font-size: 10pt;
+      height: 28px;
+      width: 186px;
+      color: #000000;
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: @jainaman224 Please refer config.scss for color coding.

File: src/backend/_scss/application.scss [hunk 4]
```diff
@@ -1302,3 +1302,14 @@ a {
   .graytext {
     color: $gray-dark;
   }
+
+  #fossasia-filter {
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: @jainaman224 Please fix indentation

File: src/backend/_scss/application.scss [hunk 5]
```diff
@@ -897,7 +897,7 @@ a {
   -webkit-transition: all 0.3s ease-out;
   -moz-transition: all 0.3s ease-out;
   transform: translate3d(0px,-200px,0px);
-  -webkit-transform: translate3d(0px,-200px,0px); 
+  -webkit-transform: translate3d(0px,-200px,0px);
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: This is important @jainaman224

File: src/backend/_scss/application.scss [hunk 6]
```diff
@@ -445,23 +444,23 @@ a {
...
   word-wrap: break-word;
   text-align: justify;
   font-size: 22px;
-} 
+}
 .tracks {
-  min-height: 70px; 
-  border-left: 1px solid rgba(0,0,0,.10);
+  min-height: 70px;
+  border-left: 1px solid rgba(0, 0, 0, .10);
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: @jainaman224 Please refer config.scss for color coding.

File: src/backend/_scss/application.scss [hunk 7]
```diff
@@ -1160,145 +1184,162 @@ a {
...
     }
   }
 
-  .graytext {
+  .contact-details {
+    left: 40px;
+    margin-bottom: 14px;
+  }
+
+  .contact-details > .contact-methods {
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: Use nesting instead of ">" @jainaman224

File: src/backend/_scss/application.scss [hunk 8]
```diff
@@ -1160,145 +1184,162 @@ a {
...
-        span {
-          .lead {
-            display: inline-block;
-            margin-bottom: 12px;
-            font-weight: 400;
-          }
-        }
+  .classic {
+    background: $white-classic-background;
+    padding: 72px 0 36px 0;
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: Please correct this @jainaman224

File: src/backend/_scss/application.scss (hunk 1 above)
Comment (jainaman224) [inline src/backend/_scss/application.scss]: Already implemented.

File: src/backend/_scss/application.scss (hunk 2 above)
Comment (jainaman224) [inline src/backend/_scss/application.scss]: Already implemented.

File: src/backend/_scss/application.scss (hunk 3 above)
Comment (jainaman224) [inline src/backend/_scss/application.scss]: Already implemented.

File: src/backend/_scss/application.scss (hunk 4 above)
Comment (jainaman224) [inline src/backend/_scss/application.scss]: Already implemented.

File: src/backend/_scss/application.scss (hunk 6 above)
Comment (jainaman224) [inline src/backend/_scss/application.scss]: Already implemented.

File: src/backend/_scss/application.scss (hunk 8 above)
Comment (jainaman224) [inline src/backend/_scss/application.scss]: Already implemented.

File: src/backend/_scss/application.scss (hunk 5 above)
Comment (jainaman224) [inline src/backend/_scss/application.scss]: Already implemented.

**Pull request merged**